### PR TITLE
Including scc_deregistration in the zvm media migration.

### PR DESCRIPTION
**Pull request by skliu**
Optimize test coverage for default module (base) in migrations from SLE 12 SP5

**Ticket:**

- https://progress.opensuse.org/issues/131120

**Verification run:**

- [my VRs](https://openqa.suse.de/tests/overview?arch=&flavor=Migration-from-SLE12-SPx&machine=&test=&modules=&module_re=&distri=sle&version=15-SP5&build=102.1&groupid=466)

### CODE REVIEW

File: schedule/migration/s390x-zVM-Media-Upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name:           s390x-zVM-Media-Upgrade.yaml
+description:    >
+    This is for s390x zvm media upgrade migration.
```
Comment (lemon-suse) [inline schedule/migration/s390x-zVM-Media-Upgrade.yaml:3]: s/zvm/zVM

Comment (skliu) [reply schedule/migration/s390x-zVM-Media-Upgrade.yaml]: Fixed.

File: schedule/migration/s390x-zVM-Media-Upgrade.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - installation/select_patterns
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - installation/first_boot
+  - console/scc_deregistration
```
Comment (lemon-suse) [inline schedule/migration/s390x-zVM-Media-Upgrade.yaml:25]: We shouldn't include the scc_deregistration here, it is not following migration flow.

You can try below:
  - console/system_prepare
  - console/hostname
  - console/force_scheduled_tasks
  - update/patch_sle
  - shutdown/grub_set_bootargs
  - shutdown/cleanup_before_shutdown
  - shutdown/shutdown

Refer to https://openqa.nue.suse.com/tests/11164534#settings

Comment (skliu) [reply schedule/migration/s390x-zVM-Media-Upgrade.yaml]: Fixed.

**Comment by tinawang123** [comment]
LGTM

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**